src/components/Background/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
You should take a look at the changes introduced in PR #71 when upgrading `typescript` to 4.1+. We don't need to import react anymore unless we need it. Typescript, in general, we import react the following way: 
```suggestion
import * as React from "react";
```